testsuite/gnat2goto/tests/record_extension/test.out

```diff
@@ -0,0 +1,14 @@
+Standard_Output from gnat2goto record_init:
+----------At: Do_Derived_Type_Definition----------
+----------record extension unsupported----------
```

Comment:
Again, I don't think we should have unsupported features in the expected test results.

Reply:
Fixed